docs/howtos/preview_on_mobile.md

```diff
@@ -0,0 +1,48 @@
+# Preview Kolibri on a mobile device
+
+_Note: This guide focuses on Kolibri as a web app rather than the Android version of Kolibri._
+
+Some tasks may require either an actual or simulated mobile device, such as a phone or tablet.
+
+## Browser development tools
+
+Most browsers provide ways to simulate mobile devices via their development tools. These tools are generally useful for testing:
+
+- Mobile viewports
+- Network and CPU throttling
+- Touch gestures
+
+Find specific guidance for the browser you are using.
+
+## Real mobile device
+
+Since browser development tools only offer an approximation, some tasks may require you to preview Kolibri on a real mobile device.
+
+### Option 1 (recommended)
+
+1. Ensure that the mobile device you wish to use for previewing Kolibri is connected to the same local network as your computer where you run the development server.
+2. Run the development server with `yarn python-devserver` and `yarn run watch --write-to-disk`
+3. Find your computer's IP address
+4. Open a browser on the mobile device and navigate to your computer's IP address with port `:8000`, `http://x.x.x.x:8000`. There, you should see Kolibri.
```

Comment:
To look at the IP, I used `hostname -I`. I am currently using **Ubuntu 22.04.5 LTS**. The out put showed me list of all IP addresses assigned to the host's network interfaces. And the only one that worked in my phone was one starting with 192.xxx.xx.xx.

Reply:
Thank you - I think I could mention `hostname` as a possible way of finding the IP, pretty sure it will useful